unittests/test_notifications.py

```diff
@@ -55,3 +58,46 @@ def test_merge_notifications_list(self):
         self.assertEqual('slack' in merged_notifications.other, True)  # default alert from global
         self.assertEqual(len(merged_notifications.other), 3)
         self.assertEqual(merged_notifications.other, {'alert', 'mail', 'slack'})
+
+    @patch('dojo.notifications.helper.send_alert_notification', wraps=send_alert_notification)
+    def test_notifications_system_level_trump(self, mock):
+        notif_user, _ = Notifications.objects.get_or_create(user=User.objects.get(username='admin'))
+        notif_system, _ = Notifications.objects.get_or_create(user=None, template=False)
+
+        last_count = 0
+        with self.subTest('user off, system off'):
+            notif_user.user_mentioned = ()  # no alert
+            notif_user.save()
+            notif_system.user_mentioned = ()  # no alert
+            notif_system.save()
+            create_notification(event="user_mentioned", title="user_mentioned", recipients=['admin'])
+            self.assertEqual(mock.call_count, last_count + 0)
```

Comment:
The `+ 0` doesn't seem necessary here?

```suggestion
            self.assertEqual(mock.call_count, last_count)
```

Reply:
From the mathematical perspective, there is no difference. It just increases readability in the context of this unit test  - all other tests use `+ number` notation.
But if it is the only issue I can remove it.